nuxt-shopify-accounts/package.json

```diff
@@ -20,6 +20,7 @@
     "@nacelle/client-js-sdk": "^3.1.1",
     "@nuxtjs/dotenv": "^1.4.1",
     "@nuxtjs/pwa": "^3.3.5",
+    "axios": "^0.21.1",
```

Comment:
Do we need `axios`? It looks like `isomorphic-unfetch` is used instead.

Reply:
Hey Nick! Yes, it was breaking the deployment! Alex also has a PR adding Axios to the project